### ISSUE TITLE
Validation, testing, RPN explanation

### ISSUE THREAD

**Issue by kamadforge** [yuxng/SubCNN#12]
I have several simple questions:

What is the difference between subcategory-aware RPN and Faster RCNN RPN? I mean, why do we need to generate proposals for both, not only for subcategory-aware RPN? Why does Faster RCNN RPN is here, at all?

What is the different between network for validation and network for testing and what is the meaning of 6k8k?

subcategory-aware detection network for validation
./experiments/scripts/kitti_val_caffenet_rcnn_multiscale.sh $GPU_ID

subcategory-aware detection network for testing
./experiments/scripts/kitti_test_caffenet_rcnn_multiscale_6k8k.sh $GPU_ID

Kindly thank you for the explanation.

**Comment by yuxng** [comment]
Subcategory-aware RPN is a new region proposal network we proposed as in the arxiv paper. You can just use one type of region proposals but not both.

On KITTI, the annotations of the test set is not released. So I divide the training set into a train set and a validation set for experiments. Validation on KITTI means training on the train set and testing on the validation set. Testing on KITTI means training on the whole training dataset and testing on the test set.
